vpn.tf

```diff
@@ -5,11 +5,11 @@ resource "azurerm_resource_group" "vpn" {
 }
 
 # Dedicated subnet in the private vnet
-resource "azurerm_subnet" "dmz" {
-  name                 = "${azurerm_virtual_network.private.name}-dmz"
+resource "azurerm_subnet" "vpn-internal" {
+  name                 = "${azurerm_virtual_network.private.name}-vpn-internal"
```

Comment:
Why not just "vpn"?

Reply:
And where the dmz subnet is declared? (since it's a draft PR, incoming later?)

Reply:
to avoid confusion at all costs